src/scss/_sofie-utility-classes.scss

```diff
@@ -0,0 +1,251 @@
+.m-0 {
```

Comment:
I don't mind, as it would be convenient I will write them like this, for now I added it here as a test concept. The names themselves come from bootstrap 4

Reply:
We should use `xs`, `s` etc instead of using `0`, `1` etc